src/local_storage.rs

```diff
@@ -80,15 +83,15 @@ impl LocalStorage {
                     InternalError::InternalInvariantFailed
                 })
             })
-            .unwrap_or(Err(InternalError::StorageItemNotFound))
+            .unwrap_or(Err(InternalError::InternalInvariantFailed))
```

Comment:
nit: Please switch this to an `or_else` (`unwrap_or_else` or `ok_or_else`) function and add a log about what went wrong.